test/pjrt/test_mesh_service.py

```diff
@@ -12,7 +12,6 @@ def _rendezvous_static_size():
     payload = b'message %d' % xm.get_ordinal()
     return xm.rendezvous("test rendezvous", payload)
 
-  @unittest.skip("fails with functionalization")
```

Comment:
to reviewers: the change in this file is irrelevant to this PR. It's to make the CI green. We need this change probably due to I made a mistake when rebasing origin/funtionalization_dynamic_shape onto origin/functionalization. The change made here is consistent to what origin/functionalization has.